atlasdb-ete-tests/src/main/java/com/palantir/atlasdb/AtlasDbEteServer.java

```diff
@@ -121,6 +131,19 @@ private SweepTaskRunner getSweepTaskRunner(SerializableTransactionManager transa
         return new SweepTaskRunner(kvs, ts, ts, txnService, ssm, cellsSweeper);
     }
 
+    private TargetedSweeper getTargetedSweeperWithFollower() {
+        TargetedSweeper targetedSweeper = TargetedSweeper.createUninitialized(() -> true, () -> 1, 0, 0, FOLLOWER);
```

Comment:
The zeroes and trivial suppliers are somewhat meaningless here. If we're often calling `createUninitialized` with trivial parameters like these, it might be best to either:
a) add another `createUninitialized` method with just the follower, or 
b) add a builder

Reply:
OK there's at least four of these rather trivial createUninitialized usages. Time to add `createUninitialized(int numShards, Follower follower)`

Reply:
Thanks, this makes it much nicer